docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -2670,6 +2670,26 @@ creation time can be used to test if a session was resumed. Currently, clients n
 the first handshake in order to receive the `NewSessionTicket` that can be used for resumption. However, this may
 change once https://bugs.openjdk.java.net/browse/JDK-8209953[JDK-8209953] is resolved.
 
+==== Dynamic client SSLContext
```

Comment:
I think it would be good to mention that this feature can be enabled using the COMMUNITY stability level (and possibly explain how to do that or point to documentation that explains that).

Reply:
@fjuma Good point, I have updated this and linked to relevant documentation. Thank you!